subsys/net/lib/lwm2m_client_utils/lwm2m/lwm2m_firmware.c

```diff
@@ -216,6 +214,11 @@ static int firmware_request_linked_update(int instance_id)
 	return postpone_update;
 }
 
+static void on_modem_lib_init(int ret, void *ctx)
```

Comment:
This change Lines: 217-221 can be drop it is already done at fota_download_delta_modem.c